README.md

```diff
@@ -12,7 +12,8 @@
 * allow click & drag
 * allow zoom
 
-You can see the updated features of this fork on : [https://image-compare.netlify.com](https://image-compare.netlify.com)<br>
+You can see the updated features of this fork on : [https://image-compare.netlify.com](https://image-compare.netlify.com)
```

Comment:
the <br> was to move to next line without a space in-between, kinda easier for reading.

Reply:
I understand, I just replace <br> inline html with a regular line break 
All my MD files respect MD Lint and using <br> trigger this lint error : 
MD033/no-inline-html: Inline HTML [Element: br]